source/instructors/admin/integration/lti-course-copy.rst

```diff
@@ -8,38 +8,62 @@ LTI Course copy
-
-.. Note:: the screenshots below are for implementation in Canvas but other LMS systems should be similar. Refer to their documentation for more assistance
-
-First enable :ref:`LTI Constant URLs <lti-keys-and-urls-information>` for your course.
 
-1.  In your existing Codio course, enable the **Enable LTI course copy** button, and save your changes
+  <div style= " margin: 0 0 10px 20px; padding: 10px; background: #f0f0f0; border: 3px solid #00ece5;">
+  <strong>Note:</strong> The screenshots below are for implementation in Canvas but other LMS systems should be similar. Refer to their documentation for more assistance.
+  </div>
 
-.. figure:: /img/lti/enable_class_fork.png
-   :alt: Enable course copy field
 
+First enable :ref:`LTI Constant URLs <lti-keys-and-urls-information>` for your course.
 
-2.  Both custom parameters should be added to the tool in the Master course.
-3.  In your LMS "Copy this Course" (or equivalent term for your LMS) and create your new course
-
-.. figure:: /img/lti/copy_course.png
-   :alt: Copy LMS Course
 
-3.  When completed go to **External Apps** and edit the existing app connecting Codio to your LMS
-4.  Replace the existing custom field that was set from your original Codio course with an id of your own (e.g something like codio_course_target_id=semester-year)
 
-.. Note:: If your LMS supports ``lis_course_offering_sourcedid`` you do not need to specify ``codio_course_target_id`` for the copied course, ``lis_course_offering_sourcedid`` will be used automatically to detect the course.  For cases when ``lis_course_offering_sourcedid`` is needed for course copy but not available due to privacy settings in Canvas, a custom parameter ``custom_codio_course_offering_sourcedid=$CourseOffering.sourcedId`` can be tried.
 
-.. figure:: /img/lti/parent_class.png
-   :width: 400
-   :alt: Parent course ID
++-----------------------------------------------+------------------------------------------------------------------------------+
+| 1. In your existing Codio course, enable      | .. figure:: /img/lti/enable_class_fork.png                                   |
+|    the **Enable LTI course copy** button,     |    :alt: Enable course copy field                                            |
+|    and save your changes.                     |                                                                              |
+|                                               |                                                                              |
+| |                                             |                                                                              |
+|                                               |                                                                              |
+| 2. Both custom parameters should be added     |                                                                              |
```

Comment:
For the sentence below:
Both custom parameters should be added to the tool in the Master course.

I would add one clarification:
Both custom parameters should be added to the tool in the Master course in Canvas.